src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static core.basesyntax.FileCreator.createFile;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
```

Comment:
Don't begin class or method implementation with empty line. Extra empty lines detected at the beginning of the class and main method. Please remove the redundant empty lines (lines 17, 22, 26, 34, 38, and 42).